README.md

```diff
@@ -2,6 +2,6 @@
 webArchiver is a project in alpha with the main purpose of Archiving Websites but can be used to archived anything.   
 It saves space by dealing with duplicates, keeps all files in a central location and keeps logs.   
 ##### Other new functions like cgi web interfaces, automatic archiving of websites and other terminal functions are comming!   
```

Comment:
##### Other new functions like CGI web interfaces, automatic archiving of websites and other terminal functions are coming!